### PR TITLE
Layout & formatting of example descriptions on FHIR Artefacts page

### PR DESCRIPTION
**Pull request by dbojicic**
Improvement of layout and formatting example descriptions on FHIR Artefacts page.
This PR does not include in scope improving the content of the descriptions.

### CODE REVIEW

**Comment by dt-r** [comment]
@dbojicic this will be reviewed in parallel with the equivalent PR for AU Core. Until that PR is available this is on hold.

File: input/examples/organization-elimbah-medical-center.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
         <valueString value="Organization - Elimbah Medical Center"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="Elimbah Medical Center"/>
+        <valueMarkdown value="This is an organization example using the *AU Core Organization* profile."/>
```
Comment (dt-r) [inline input/examples/organization-elimbah-medical-center.xml:9]: Insufficient, provide information on the organization example.

File: input/examples/patient-fred-roberts.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
         <valueString value="Patient - Fred Roberts"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="Fred Roberts"/>
+        <valueMarkdown value="This is a patient example using the *AU Core Patient* profile."/>
```
Comment (dt-r) [inline input/examples/patient-fred-roberts.xml:9]: Insufficient, provide information on the patient example. This project task isn't about better grammar it's about layout information for the reader.

File: input/examples/order-xray-1.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
     <valueString value="ServiceRequest - X-Ray"/>
   </extension>
   <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-    <valueMarkdown value="X-Ray order for patient Fred Roberts"/>
+    <valueMarkdown value="This is an example of *AU eRequesting Imaging Request* profile, illustrating a request for an X-Ray for a patient. &#xa;&#xa;Patient: Fred Roberts."/>
```
Comment (dt-r) [inline input/examples/order-xray-1.xml:8]: Insufficient, provide more information on the example.

File: input/examples/order-fbc-1.xml [hunk 4]
```diff
@@ -5,7 +5,7 @@
         <valueString value="ServiceRequest - Full Blood Count"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="FBC order for patient Fred Roberts"/>
+        <valueMarkdown value="This is an example of the *AU eRequesting Pathology Request* profile, illustrating a request for a FBC for a patient. &#xa;&#xa;Patient: Fred Roberts."/>
```
Comment (dt-r) [inline input/examples/order-fbc-1.xml:8]: Insufficient provide more information on the example.

File: input/examples/encounter-consultation.xml [hunk 5]
```diff
@@ -4,7 +4,7 @@
         <valueString value="Encounter - Consultation"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="Consultation with patient Fred Roberts"/>
+        <valueMarkdown value="This is a consultation encounter example using the *AU Core Encounter* profile.&#xa;&#xa;Patient: Fred Roberts"/>
```
Comment (dt-r) [inline input/examples/encounter-consultation.xml:7]: Insufficient, provide more information on the encounter.

File: input/examples/practitioner-guthridge-jarred.xml [hunk 6]
```diff
@@ -6,7 +6,7 @@
         <valueString value="Practitioner - Dr. Jarred Guthridge"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="Dr. Jarred Guthridge"/>
+        <valueMarkdown value="This is a practitioner example using the *AU Core Practitioner* profile."/>
```
Comment (dt-r) [inline input/examples/practitioner-guthridge-jarred.xml:9]: Provide more info.

File: input/examples/organization-elimbah-medical-center.xml [hunk 7]
```diff
@@ -6,7 +6,7 @@
...
     </extension>
```
Comment (dt-r) [inline input/examples/organization-elimbah-medical-center.xml:10]: For all AU Core profiles don't say "using the ..." have same text as eRequesting "of ..."

File: input/examples/practitionerrole-guthridge-jarred-gp.xml [hunk 8]
```diff
@@ -6,7 +6,7 @@
         <valueString value="PractitionerRole - General practitioner"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="Dr Jarred Guthridge, GP at Elimbah Medical Center"/>
+        <valueMarkdown value="This is general practitioner example using the *AU Core PractitionerRole* profile."/>
```
Comment (dt-r) [inline input/examples/practitionerrole-guthridge-jarred-gp.xml:9]: Not enough information on the example.

File: input/examples/encounter-consultation.xml (hunk 5 above)
Comment (dt-r) [inline input/examples/encounter-consultation.xml:7]: Don't linebreak Patient. Apply to all example markdown.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments, noting given timeframes (this needs to be merged today) and priorities it may be better to:
* address only consistency and formatting. 
* reframe this PR as layout & formatting rather than substantial content improvements to the descriptions. 
* move the project task into the backlog to come back and improve these descriptions post ballot snapshot. 

I will leave that to your discretion.

File: input/examples/encounter-consultation.xml (hunk 5 above)
Comment (dbojicic) [reply input/examples/encounter-consultation.xml:7]: Changed

File: input/examples/organization-elimbah-medical-center.xml (hunk 7 above)
Comment (dbojicic) [reply input/examples/organization-elimbah-medical-center.xml:10]: Fixed

**Comment by dbojicic** [comment]
Addressed the formatting and consistency issues above. 
Will move the task to the backlog to address post ballot snapshot.

File: input/examples/encounter-consultation.xml [hunk 9]
```diff
@@ -4,7 +4,7 @@
         <valueString value="Encounter - Consultation"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="This is a consultation encounter example using the *AU Core Encounter* profile.&#xa;&#xa;Patient: Fred Roberts"/>
+        <valueMarkdown value="This is a consultation encounter example for the *AU Core Encounter* profile."/>
```
Comment (dt-r) [inline input/examples/encounter-consultation.xml:7]: Missing the Patient for this example

Comment (dt-r) [inline input/examples/encounter-consultation.xml:7]: For all examples, to be consistent with AU Core:
1. Start with "Shows an example of <insert thing here> for the *<insert profile* profile. Patient <insert patient>."

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments

File: input/examples/encounter-consultation.xml (hunk 9 above)
Comment (dbojicic) [reply input/examples/encounter-consultation.xml:7]: Added

Comment (dbojicic) [reply input/examples/encounter-consultation.xml:7]: Reformatted

Comment (dt-r) [reply input/examples/encounter-consultation.xml]: Some were, some aren't

File: input/examples/order-xray-1.xml [hunk 10]
```diff
@@ -5,7 +5,7 @@
     <valueString value="ServiceRequest - X-Ray"/>
   </extension>
   <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-    <valueMarkdown value="This is an example for the *AU eRequesting Imaging Request* profile, illustrating a request for an X-Ray for a patient. Patient: Fred Roberts."/>
+    <valueMarkdown value="Shows an example for the *AU eRequesting Imaging Request* profile, illustrating a request for an X-Ray for a patient. Patient: Fred Roberts."/>
```
Comment (dt-r) [inline input/examples/order-xray-1.xml:8]: Reformat this one.

File: input/examples/order-fbc-1.xml [hunk 11]
```diff
@@ -5,7 +5,7 @@
         <valueString value="ServiceRequest - Full Blood Count"/>
     </extension>
     <extension url="http://hl7.org/fhir/StructureDefinition/resource-instance-description">
-        <valueMarkdown value="This is an example for the *AU eRequesting Pathology Request* profile, illustrating a request for a FBC for a patient. Patient: Fred Roberts."/>
+        <valueMarkdown value="Shows an example for the *AU eRequesting Pathology Request* profile, illustrating a request for a FBC for a patient. Patient: Fred Roberts."/>
```
Comment (dt-r) [inline input/examples/order-fbc-1.xml:8]: Reformat this one.

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments.

File: input/examples/order-xray-1.xml (hunk 10 above)
Comment (dbojicic) [reply input/examples/order-xray-1.xml]: Fixed.

File: input/examples/order-fbc-1.xml (hunk 11 above)
Comment (dbojicic) [reply input/examples/order-fbc-1.xml]: Fixed.

**Review by dt-r — APPROVED** [review]

**Pull request merged**